### PR TITLE
Buffs the greedy god vending machine

### PR DESCRIPTION
**Pull request by bob-b-b**
## About The Pull Request
The greedy god vending machine was pretty underwhelming compared to normal custom vending machines, it only had fancy light beams, this makes it harder to break and lets it store/sell more items

## Why It's Good For The Game

Makes underused greedy god deity more appealing by increasing the 1000 credit vending machines stats and capacity

## Changelog

:cl:
balance: Increased greed vending machine integrity and max items
/:cl:

### CODE REVIEW

File: code/modules/vending/_vending.dm [hunk 1]
```diff
@@ -1340,6 +1340,8 @@ GLOBAL_LIST_EMPTY(vending_products)
 	add_overlay(panel_type)
 	//and references the deity
 	name = "[GLOB.deity]'s Consecrated Vendor"
+	max_integrity = 700
+	max_loaded_items = 40
```
Comment (tralezab) [inline code/modules/vending/_vending.dm:1344]: Why on initialize instead of on declaration?

**Review by tralezab — CHANGES_REQUESTED** [review]
I am confused

File: code/modules/vending/_vending.dm (hunk 1 above)
Comment (bob-b-b) [reply code/modules/vending/_vending.dm]: Moved, my mistake

**Review by tralezab — APPROVED** [review]
shore

**Pull request merged**